mojave/items/clothing/head/helmet.dm

```diff
@@ -332,7 +332,7 @@
 
 /obj/item/clothing/head/helmet/ms13/army
 	name = "army helmet"
-	desc = "An old pre-war helmet, even by their standards. Seems to have the addition of quite a bit of ballistic padding on the inside, likely to try and modernize it."
```

Comment:
The only thing I'd have removed was 'even by their standards' and just went with the older vague initial description